eks-anywhere-common/Addons/Partner/newrelic/newrelic.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: newrelic
+  namespace: newrelic
+spec:
+  chart:
+    spec:
+      chart: nri-bundle
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: newrelic-charts
+        namespace: flux-system
+      version: 5.0.4
+  interval: 1m0s
+  targetNamespace: newrelic
+  values:
+    global:
+      cluster: 
+        valuesFrom:
```

Comment:
@anshrma This is incorrect and Deployment fails. You should have something like below at the end of the YAML:

```
  - kind: ConfigMap
    name: aws-env-metadata
    namespace: kube-system
    valuesKey: clusterName
    targetPath: global.cluster
```
Line Number 21-26 is not required.